app/coffee/Features/Announcements/AnnouncementsHandler.coffee

```diff
@@ -19,6 +19,9 @@ module.exports = AnnouncementsHandler =
 		if !user? and !user._id?
 			return callback("user not supplied")
 
+		timestamp = user._id.toString().substring(0,8)
+		userSignupDate = new Date( parseInt( timestamp, 16 ) * 1000 )
```

Comment:
This user is from the session. Wanted to avoid another call to db while this is via http and isolated.